src/containers/Accounts/AMM/AMMAccounts/AMMAccountHeader/AMMAccountHeader.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import '../../../../shared/css/nested-menu.scss'
+import '../../../AccountHeader/styles.scss'
+import '../../../AccountHeader/balance-selector.scss'
```

Comment:
@ckniffen do the CSS files need to be included here like this, or will inheritance take care of it?

Also, these should go below the JS imports

Reply:
this is not a blocker - lets skip it

Reply:
All scss styles are included globally by the build importing them wont hurt anything especially good if the other place gets changed the styles wont disappears. 

We should look into an eslint rule to enforce.